tests/devapps/TestTelemetry/Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.Metrics;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.TelemetryCore.OpenTelemetry;
+using Microsoft.Identity.Test.Integration.NetFx.Infrastructure;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+
+internal class Program
+{
+    static async Task Main(string[] args)
+    {
+        Console.WriteLine("Hello World!");
+
+        using var meterProvider = Sdk.CreateMeterProviderBuilder()
+            .AddMeter("MSAL.Net.Meter")
```

Comment:
Does it still work considering that this name doesn't match the name on the actual meter?

Reply:
Missed to check in the updated file. It will not work with another name.

Reply:
@neha-bhargava still not fixed?

Reply:
Missed. Thanks for pointing out